src/main/java/org/cancogenvirusseq/muse/model/DownloadAnalysisFetchResult.java

```diff
@@ -1,14 +1,13 @@
 package org.cancogenvirusseq.muse.model;
 
+import java.util.Optional;
+import java.util.UUID;
 import lombok.Getter;
 import org.cancogenvirusseq.muse.model.song_score.Analysis;
 import org.cancogenvirusseq.muse.model.song_score.AnalysisFile;
 import org.cancogenvirusseq.muse.model.song_score.SongScoreServerException;
 import org.springframework.http.HttpStatus;
 
-import java.util.Optional;
-import java.util.UUID;
-
```

Comment:
run a quick `mvn:fmt` and commit so we can eliminate this formatting noise on PR